cairo-libs/packages/ics23/src/specs.cairo

```diff
@@ -34,7 +34,7 @@ pub fn tendermint_spec() -> ProofSpec {
         min_prefix_length: 1,
         max_prefix_length: 1,
         child_size: 32,
-        empty_child: "",
```

Comment:
Source for these values.

https://github.com/cosmos/ics23/blob/a324422529b8c00ead00b4dcee825867c494cddd/rust/src/api.rs#L332

Reply:
[f74daee](https://github.com/informalsystems/ibc-starknet/pull/305/commits/f74daee8bead51626ebffca6b408695b94fd622c)